htdocs/public/eventorganization/attendee_new.php

```diff
@@ -268,6 +268,7 @@ function llxFooterVierge()
 		} else {
 			// Need to create a confattendee
 			$confattendee->date_creation = dol_now();
+			$confattendee->date_subscription = dol_now();
```

Comment:
The date_subscription is not a mandatory field. It is field when we record a payment (a different step than recording an attendee, coming after).